src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/FrozenHashTable.cs

```diff
@@ -186,13 +186,14 @@ private static int CalcNumBuckets(ReadOnlySpan<int> hashCodes, bool optimizeForR
             // In our precomputed primes table, find the index of the smallest prime that's at least as large as our number of
             // hash codes. If there are more codes than in our precomputed primes table, which accommodates millions of values,
             // give up and just use the next prime.
+            ReadOnlySpan<int> primes = HashHelpers.Primes;
             int minPrimeIndexInclusive = 0;
-            while (minPrimeIndexInclusive < HashHelpers.s_primes.Length && codes.Count > HashHelpers.s_primes[minPrimeIndexInclusive])
+            while ((uint)minPrimeIndexInclusive < (uint)primes.Length && codes.Count > primes[minPrimeIndexInclusive])
```

Comment:
You might not need the `uint` trick here anymore.

We updated `Span.Length` and `ROSpan.Length` to be tracked as "never negative" a couple weeks back.

Reply:
Thanks.  We appear to have a few hundred of these, still (though some are strings and arrays... just doing a regex search makes it hard to discern).  I'll leave this as-is and we can subsequently clean them all up if they're unnecessary for codegen.